hail/python/hailtop/batch/backend.py

```diff
@@ -452,6 +452,26 @@ def __init__(self,
         gcs_kwargs = {'project': google_project}
         self.__fs: RouterAsyncFS = RouterAsyncFS(default_scheme='file', gcs_kwargs=gcs_kwargs)
 
+        self.regions: Optional[List[str]]
+        if isinstance(regions, str):
+            self.regions = [regions]
+        elif regions is None:
+            _regions = configuration_of('batch', 'regions', regions, None)
+            if _regions is not None:
+                self.regions = _regions.split()
+            else:
+                self.regions = None
+                warnings.warn('No default cloud regions have been specified.\n'
```

Comment:
We should give a long hard careful thought to whether or not we want this to be a warning. I think this UX post covers some of the issues with warnings. https://ux.stackexchange.com/questions/44609/how-do-i-avoid-users-becoming-numb-to-warnings

I'm hesitant to add yet another piece of visual noise that users will mostly ignore. This also will trigger a warning even if users *are* specifying regions for every job (for example, they have data in different regions, so it doesn't make sense to specify at batch-level, instead they specify at the job-level).

Reply:
Also consider: we don't and never have warned users when they read data in a regional bucket from the wrong region. That has always been a problem and we have always just expected users to handle that. Why is this situation different?

Reply:
I'm fine removing this. I was trying to be extra clear to reduce bad experiences for users based on the recent change if they missed the announcement messages, but you're right that we don't need to do this.